resources_pages/install_ds_stack_windows.md

```diff
@@ -3,290 +3,540 @@ layout: page
-1.36.1
-2213894ea0415ee8c85c5eea0d0ff81ecc191529
-x64
+# Add R and Rscript to PATH
+export PATH="/c/Program Files/R/R-4.0.2/bin/x64":$PATH
 ```
 
-### Customizing VS Code with Extensions
+> Note: Although it is possible to install R through Anaconda, we highly recommend not doing so. In case you have already installed R using Anaconda you can remove it by executing `conda uninstall r-base`.
 
-The real magic of VS Code is in the extensions that let you add languages, debuggers, and tools to your installation to support your specific workflow. From within VS Code you can open up the [Extension Marketplace](https://code.visualstudio.com/docs/editor/extension-gallery) to browse and install extensions by clicking on the Extensions icon in the Activity Bar indicated in the figure below.
+### RStudio
 
-![](/resources_pages/imgs/vscode.png)
+Download the Windows version of RStudio from <https://www.rstudio.com/products/rstudio/download/preview>. Open the file and follow the installer instructions.
 
-To install an extension, you simply search for it in the search bar, click the extension you want, and then click "Install". There are extensions available to make almost any workflow or task you are interested in more efficient! Here we are interested in setting up VS Code as a Python IDE. To do this, search for and install the following extensions:
+To see if you were successful, try opening RStudio by clicking on its icon. It should open and looks something like this picture below:
 
-- Python (everything Python: notebooks, debugging, linting, formatting, etc.)
-- markdownlint (markdown linting and style checking extension)
-- GitLens (powerful extension that extends VS Code's native git capabilities)
-- Docker (easily use Docker from VS Code)
-- (Optional) Material Theme and/or Predawn Theme Kit (additional colour themes to choose from)
-- (Optional) Material Icon Theme (great-looking custom file icons!)
-- (Optional) Bracket Pair Colorizer 2 (add colour to help distinguish your brackets: (), [], {})
+![](/resources_pages/imgs/RStudio.png)
 
-This [video tutorial](https://www.youtube.com/watch?v=-nh9rCzPJ20) is an excellent introduction to using VS Code in Python.
+By default, RStudio installs packages to `C:/Users/your_username/Documents/R/win-library/4.0`, but when it is run directly from the terminal it instead uses  `C:/Users/your_username/R/win-library/4.0`. To make sure that R from terminal uses the same package directory as R studio, we will edit `~/.Rprofile` by typing:
+
+```
+code ~/.Rprofile
+```
+
+And adding the following
+
+```
+# Make R from terminal use the same package directory as RStudio
+.libPaths(c(paste("C:/Users/", Sys.getenv("USERNAME"), "/Documents/R/win-library/4.0" , sep=''), .libPaths()))
+```
+
+### Essential R packages
+
+Next, install the key R packages needed for the start of MDS program,
+by typing the following into the R terminal in RStudio:
+
+```
+install.packages(c('tidyverse', 'blogdown', 'xaringan', 'renv'))
```

Comment:
There were lots of more packages in the orignal windows install, I changed to jsut the ones from Ububntu and Mac, but if we need some of the other ones, let me know and I will add them back.

Reply:
These were the original ones 

```
install.packages(c('repr', 'IRdisplay', 'evaluate', 'crayon', 'pbdZMQ', 'devtools', 'uuid', 'digest', 'tidyverse'))
```

Reply:
Ahhh, I think this is the order in which we have changed the install instructions... 'repr', 'IRdisplay', 'evaluate', 'crayon', 'pbdZMQ', 'uuid', and 'digest'' are all dependencies of the `IRKernel` package... (see: https://github.com/IRkernel/IRkernel/blob/9871fc1aafc7b8da34b81c8511606d596b9aa99c/DESCRIPTION#L25).

Reply:
So I don't think they are needed, they should be installed later as dependencies when the  `IRKernel` package is installed.

Reply:
Good to know, I didn't run into any problems with IRkernel not finding packages when running from Jupyter, so if these are not used foranything else we should be good